monkey/monkey_island/cc/ui/src/components/ui-components/utils/GridNoRowsOverlay.js

```diff
@@ -68,7 +70,7 @@ const CustomNoRowsOverlay = () => {
           </g>
         </g>
       </svg>
-      <Box sx={{mt: 1}}>No Rows</Box>
+      <Box sx={{mt: 1}}>{message || DEFAULT_MESSAGE}</Box>
```

Comment:
We should change the image as well

Reply:
Let's do it in the future as it involves changing the data grid.